### DISCUSSION TITLE
Change name from go-micro to go-nitro :D

### DISCUSSION THREAD

**Discussion by asim** [micro/go-micro#2067] [Q&A] [answered]
Votes to change the name from go-micro to go-nitro 😄 

https://aws.amazon.com/blogs/aws/aws-nitro-enclaves-isolated-ec2-environments-to-process-confidential-data/

**Comment by scorsi** [comment]
Honestly, it will help better distinguish between micro and go-micro :)

**Comment by asim** [comment]
Well I have just acquired go-nitro.dev and go-nitro.com. So perhaps this will happen... it is sad to leave go-micro behind, but all things must under go some for of evolution, even if that means naming.

**Comment by asim** [comment, marked as answer]
Because I waste no time. Go Micro is now Nitro.

**Reply by scorsi** [reply]
Haha was so fast !

**Reply by asim** [reply]
Gotta get it done!

**Reply by GuodongFan** [reply]
? any change?

**Comment by jolee11** [comment]
I really like this change!

**Comment by asim** [comment]
5 years ago
....
